web/src/app/workplace/models/task.ts

```diff
@@ -4,44 +4,44 @@ import {ObjectReference} from './object-reference';
 
 export class Task {
   constructor(public taskId: string,
-              public primaryObjRef: ObjectReference = undefined,
-              public workbasketSummaryResource: Workbasket = undefined,
-              public classificationSummaryResource: Classification = undefined,
-              public businessProcessId: string = undefined,
-              public parentBusinessProcessId: string = undefined,
-              public owner: string = undefined,
-              public created: string = undefined,    // ISO-8601
-              public claimed: string = undefined,    // ISO-8601
-              public completed: string = undefined,  // ISO-8601
-              public modified: string = undefined,   // ISO-8601
-              public planned: string = undefined,    // ISO-8601
-              public due: string = undefined,        // ISO-8601
-              public name: string = undefined,
-              public creator: string = undefined,
-              public description: string = undefined,
-              public note: string = undefined,
-              public state: any = undefined,
-              public read: boolean = undefined,
-              public transferred: boolean = undefined,
-              public priority: number = undefined,
+              public primaryObjRef: ObjectReference = new ObjectReference(),
+              public workbasketSummaryResource: Workbasket = null,
+              public classificationSummaryResource: Classification = null,
+              public businessProcessId: string = '',
+              public parentBusinessProcessId: string = '',
+              public owner: string = '',
+              public created: string = '',    // ISO-8601
+              public claimed: string = '',    // ISO-8601
+              public completed: string = '',  // ISO-8601
+              public modified: string = '',   // ISO-8601
+              public planned: string = '',    // ISO-8601
+              public due: string = '',        // ISO-8601
```

Comment:
is this intended as empty string and not null?

Reply:
I corrected it